.github/workflows/trivy.yml

```
@@ -28,9 +28,45 @@ on:
 
 env:
   IMAGE_NAMESPACE: 'tractusx'
+  DATASPACE_SDK_IMAGE_NAME: "dataspace-sdk"
   INDUSTRY_SDK_IMAGE_NAME: "industry-sdk"
 
 jobs:
+  analyze-dataspace-sdk:
+    name: Analyze Dataspace SDK
+    runs-on: ubuntu-latest
+    permissions:
+      actions: read
+      contents: read
+      security-events: write
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+
+      - name: Build dataspace-sdk Docker image
+        id: build-docker-dataspace-sdk
+        run: |
+            cd tractusx_sdk/dataspace
```

```suggestion
            cd dataspace
```
The same in the line 85 "cd industry"